### DISCUSSION TITLE
Different behaviour if there are no page labels?

### DISCUSSION THREAD

**Discussion by phkus** [RyotaUshio/obsidian-pdf-plus#141] [Q&A] [answered]
If you have something like the default "p.{{pageLabel}}" as display text, the result will look the same whether or not the PDF has added page labels. The problem is that in one of these cases this is a page reference that I can cite, in the other case it's not. Is there a way to distinguish these two?

If pageLabel is the same as page: "PDF p.{{pageLabel}}"
If pageLabel is different: "p.{{pageLabel}}"

The reason I'm asking is that the docs mention somewhere that these are just JS expressions, so it may be possible to include an if-statement. But I don't know how to do this specifically, because the part that I would want to change (the "p.") is outside of the {{...}}.

**Comment by RyotaUshio** [comment, marked as answer]
You can use the [conditional operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_operator) and [template strings](https://javascript.info/string#:~:text=Backticks%2C%20however%2C%20allow%20us%20to%20embed%20any%20expression%20into%20the%20string%2C%20by%20wrapping%20it%20in%20%24%7B%E2%80%A6%7D%3A).

```
{{ page == pageLabel ? `PDF p.${pageLabel}` : `p.${pageLabel}` }}
```

**Reply by phkus** [reply]
Yes that works, thanks for your help and for the great plugin!
